cloudformation/membership-attribute-service.yaml

```diff
@@ -135,7 +136,8 @@ Resources:
             - dynamodb:BatchGetItem
             - dynamodb:DescribeTable
             Resource:
-              Fn::FindInMap: [ StageVariables, { Ref: Stage }, DynamoDBTableTestUsers ]
+              Fn::ImportValue:
```

Comment:
On second thoughts, let's leave the permissions to read/write to the old tables alone. We can remove these when we are happy that we don't need to roll this change back (i.e. when we delete the old, unencrypted tables).